runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -74,6 +74,12 @@
 	private final JLabel expHour = new JLabel();
 	private final JLabel expLeft = new JLabel();
 	private final JLabel actionsLeft = new JLabel();
+	private final JMenuItem openXpTracker = new JMenuItem("Open online tracker");
```

Comment:
Any reason why those were extracted from constructor when they are used only there?

Reply:
No functional reason at least.

When I added one of the menu items to it, I felt it was inconsistent to have only some of the menu items be inside the constructor and some outside.

Reply:
I believe best practice is to reduce scope and visibility as much as possible and even when it is inconsistent, this is doing opposite

Reply:
Reverted.